src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -460,27 +460,11 @@ def validate_dry_run(namespace):
         raise MutuallyExclusiveArgumentError("The '--dry-run' and '--yes' options cannot be specified together.")
 
 
-def validate_kv_revision_retention_days(namespace):
-    if namespace.kv_revision_retention_days is None:
+def validate_kv_revision_retention_period(namespace):
+    if namespace.kv_revision_retention_period is None:
         return
 
-    retention_days = int(namespace.kv_revision_retention_days)
+    retention_period = int(namespace.kv_revision_retention_period)
```

Comment:
Is this line needed since we will the `type` specified when defining the argument guarantees this will be an int?